docker/master/Dockerfile

```diff
@@ -3,8 +3,6 @@ FROM noaagsl/slurm-master:latest
 RUN apt-get update -y && apt-get install -y \
     libpmi2-0-dev
 
-# Set up user shell init
-RUN echo "source /opt/intel/oneapi/setvars.sh > /dev/null 2>&1" >> /home/admin/.bash_profile \
- && echo ". /opt/spack/share/spack/setup-env.sh" >> /home/admin/.bash_profile \
- && echo "spack env activate flux" >> /home/admin/.bash_profile \
- && chown admin:admin /home/admin/.bash_profile
+# Set up Intel and Spack initialization
+RUN echo "source /opt/intel/oneapi/setvars.sh > /dev/null 2>&1" > /etc/profile.d/intel-oneapi.sh \
+ && echo "source /opt/etc/activate-parsl-flux.sh" > /etc/profile.d/parsl-flux.sh
```

Comment:
The comment here says it's setup Spack initialization. This to me looks like it's activating the parsl-flux initialization? 

Should we be using `. /opt/spack/share/spack/setup-env.sh \` to set up the spack initialization? Do we need to update this comment?

Reply:
Yes, that comment (and the others) could be more precise.